Frameworks/include/CoreTextInternal.h

```diff
@@ -48,28 +48,13 @@ inline void _SafeRelease(T** p) {
 
 CFAttributedStringRef _CTTypesetterGetAttributedString(CTTypesetterRef typesetter);
 
-@interface _CTRun : NSObject {
-@public
-    StrongId<NSMutableDictionary<NSString*, id>> _attributes;
-    CFRange _range;
-    StrongId<NSString> _stringFragment;
-    DWRITE_GLYPH_RUN _dwriteGlyphRun;
-    std::vector<CFIndex> _stringIndices;
-    CGFloat _relativeXOffset;
-    CGFloat _relativeYOffset;
-    std::vector<CGSize> _glyphAdvances;
-    std::vector<CGPoint> _glyphOrigins;
-    CGAffineTransform _textMatrix;
-}
-@end
-
 @interface _CTLine : NSObject {
 @public
     CFRange _strRange;
     CGFloat _relativeXOffset;
     CGFloat _width;
     NSUInteger _glyphCount;
-    StrongId<NSMutableArray<_CTRun*>> _runs;
+    woc::StrongCF<CFMutableArrayRef> _runs;
```

Comment:
not really, I don't want to do extra work to make the usage of _runs work and just kill it in CTLine :p

---
In reply to: [111455578](https://github.com/Microsoft/WinObjC/pull/2511#discussion_r111455578) [](ancestors = 111455578)